src/main/java/frc/robot/SpeedLookupTable.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot;
+
+import java.util.HashMap;
+
+public class SpeedLookupTable {
+  static SpeedLookupTable mInstance;
+  public static synchronized SpeedLookupTable getInstance() {
+    if (mInstance == null) {
+      mInstance = new SpeedLookupTable();
+    }
+    return mInstance;
+  }
+
+  // the key is distance in meters
+  // the value is velocity in whatever our system uses
+  // note: This MUST be sorted by lowest distance first
+  double[][] map = {
```

Comment:
This is a member variable. Suggesting a more meaningful name `mInterpolationMap`

Reply:
Fixed